### PR TITLE
surface schema location in deployed backend client

### PR DESCRIPTION
**Pull request by sobolk**
## Problem

Console would like to know where Schema is located.

**Issue number, if available:**

## Changes

Add already existing output that contains schema url to deployed backend client output.

**Corresponding docs PR, if applicable:**

## Validation

Unit test.

## Checklist

- [ ] If this PR includes a functional change to the runtime behavior of the code, I have added or updated automated test coverage for this change.
- [ ] If this PR requires a change to the [Project Architecture README](../PROJECT_ARCHITECTURE.md), I have included that update in this PR.
- [ ] If this PR requires a docs update, I have linked to that docs PR above.
- [ ] If this PR modifies E2E tests, makes changes to resource provisioning, or makes SDK calls, I have run the PR checks with the `run-e2e` label set.

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

File: packages/deployed-backend-client/src/deployed_backend_client.test.ts [hunk 1]
```diff
@@ -258,6 +258,7 @@ void describe('Deployed Backend Client', () => {
         additionalAuthTypes: [],
         conflictResolutionMode: undefined,
         apiId: 'awsAppsyncApiId',
+        modelSchemaS3Uri: 's3://bucketName/model-schema.graphql',
```
Comment (cwoolum) [inline packages/deployed-backend-client/src/deployed_backend_client.test.ts:261]: Can we make this just the bucket name or ARN? We actually need a JSON file from that bucket that doesn't exist yet.

Comment (sobolk) [reply packages/deployed-backend-client/src/deployed_backend_client.test.ts:261]: Synced offline. We'll expose URI to requested file.

**Comment by sobolk** [comment]
Closing this PR. We'll take another route.

**Comment by sobolk** [comment]
> Closing this PR. We'll take another route.

Reopening, seems that URI to schema is desired solution.

**Review by edwardfoyle — APPROVED** [review]

**Pull request merged**